opm/simulators/linalg/PreconditionerFactory.hpp

```diff
@@ -143,12 +143,33 @@ class PreconditionerFactory
         using SmootherArgs = typename Dune::Amg::SmootherTraits<Smoother>::Arguments;
         SmootherArgs smootherArgs;
         smootherArgs.iterations = prm.get<int>("iterations", 1);
+	//int iluwitdh = prm.get<int>("iluwidth", 0);
+	//MILU_VARIANT milu = convertString2Milu(prm.get<std::string>("milutype",std::string("ilu")));
+	//smootherArgs.setMilu(milu);
+	//smootherArgs.setN(iluwitdh);
         // smootherArgs.overlap=SmootherArgs::vertex;
         // smootherArgs.overlap=SmootherArgs::none;
         // smootherArgs.overlap=SmootherArgs::aggregate;
-        smootherArgs.relaxationFactor = prm.get<double>("relaxation", 0.9);
+        smootherArgs.relaxationFactor = prm.get<double>("relaxation", 1.0);
         return smootherArgs;
     }
+    // template <>
+    // static auto amgSmootherArgs< Opm::ParallelOverlappingILU0< Matrix, Vector, Vector, Comm>  >(const boost::property_tree::ptree& prm)
+    // {
+    // 	using Smoother = Opm::ParallelOverlappingILU0<Matrix, Vector, Vector, Comm>;
+    //     using SmootherArgs = typename Dune::Amg::SmootherTraits<  Smoother >::Arguments;
+    //     SmootherArgs smootherArgs;
+    //     smootherArgs.iterations = prm.get<int>("iterations", 1);
+    // 	int iluwitdh = prm.get<int>("iluwidth", 0);
+    // 	MILU_VARIANT milu = convertString2Milu( prm.get<std::string>("milutype",std::string("ilu") ) );
+    // 	smootherArgs.setMilu(milu);
+    // 	smootherArgs.setN(iluwitdh);
+    //     // smootherArgs.overlap=SmootherArgs::vertex;
+    //     // smootherArgs.overlap=SmootherArgs::none;
+    //     // smootherArgs.overlap=SmootherArgs::aggregate;
+    //     smootherArgs.relaxationFactor = prm.get<double>("relaxation", 1.0);
+    //     return smootherArgs;
+    // }
```

Comment:
Why the addition a a completely commented out code block?

Reply:
Because It should be fixed. But commenting out the code do not compile at least for me.

Reply:
Sorry, but that is not an argument. Can't you use git stash?